app/controllers/page_views_controller.rb

```diff
@@ -64,4 +64,14 @@ def update_organic_page_views
       .where("created_at > ?", 1.month.ago).sum(:counts_for_number_of_views)
     @article.update_column(:organic_page_views_past_month_count, organic_count_past_month_count)
   end
+
+  def skip_page_view_update?
+    # We don't need to update the article page views every time.
+    rand(15) != 1
```

Comment:
I think we can do better here than just random updates but I don't have a solution in mind at the moment. Just wanted to put it out there though this is on my radar as something we could improve.